G4_DailyReport/src/main/java/com/example/G4_DailyReport/service/DepartmentService.java

```diff
@@ -2,40 +2,21 @@
 
 import com.example.G4_DailyReport.model.Department;
 import com.example.G4_DailyReport.model.User;
-import com.example.G4_DailyReport.repository.DepartmentRepository;
 import com.example.G4_DailyReport.repository.UserRepository;
-import jakarta.persistence.criteria.CriteriaQuery;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.data.domain.*;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Service;
 
-import java.time.LocalDate;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
 import java.util.UUID;
-import java.util.stream.IntStream;
 
 @Service
 public class DepartmentService {
     @Autowired
-    private DepartmentRepository departmentRepository;
+    private com.example.G4_DailyReport.repository.DepartmentRepository departmentRepository;
```

Comment:
```suggestion
    private DepartmentRepository departmentRepository;
```
import đi em, cần j đưa code full thế này

Reply:
Em dùng intellij nó cứ tự import kiểu này, toàn phải sửa lại bằng tay😂